### ISSUE TITLE
Cannot remove/delete/trash a secret

### ISSUE THREAD

**Issue by ymettier** [woodpecker-ci/woodpecker#1937]
### Component

web-ui

### Describe the bug

Create a secret (I guess the name is important for this bug).

Then try to remove it with the trash button.

![image](https://github.com/woodpecker-ci/woodpecker/assets/2285664/a31a2c1f-111a-42cd-84eb-c216c26613a5)

Note : I managed to remove another secret. I think that there is a problem with the name of the secret I cannot remove.

### System Info

```shell
{"source":"https://github.com/woodpecker-ci/woodpecker","version":"next-94f2331a"}

Backend Kubernetes
```

### Validations

- [X] Read the [Contributing Guidelines](https://github.com/woodpecker-ci/woodpecker/blob/master/CONTRIBUTING.md).
- [X] Read the [docs](https://woodpecker-ci.org/docs/intro).
- [X] Check that there isn't [already an issue](https://github.com/woodpecker-ci/woodpecker/issues) that reports the same bug to avoid creating a duplicate.
- [X] Checked that the bug isn't fixed in the `next` version already [https://woodpecker-ci.org/faq#which-version-of-woodpecker-should-i-use]
- [X] Check that this is a concrete bug. For Q&A join our [Discord Chat Server](https://discord.gg/fcMQqSMXJy) or the [Matrix room](https://matrix.to/#/#woodpecker:matrix.org).

**Comment by 6543** [comment]
my guess: we just need to url-encode the name and decode it on the server side ... I did had similar bugs fixed in the past :D (gitea) - the problem is likely the `/`

**Comment by ymettier** [comment]
I confirm the bug is fixed with version `next-1d6b073ca5`.
Thanks.
